Jenkinsfile

```diff
@@ -175,12 +184,16 @@ pipeline {
           steps {
             node('built-in') {
               script{
-                sh 'newman run /var/lib/jenkins/iudx/cat/Newman/iudx-catalogue-server-v5.0.0.postman_collection.json -e /home/ubuntu/configs/cd/cat-postman-env.json --delay-request 1000 --insecure -r htmlextra --reporter-htmlextra-export /var/lib/jenkins/iudx/cat/Newman/report/cd-report.html --reporter-htmlextra-skipSensitiveData'
+            sh 'mvn test-compile failsafe:integration-test -DskipUnitTests=true'
```

Comment:
Here as well, since we want to test the server deployed in the dev swarm, we need to test against that server by giving remote host `api.cat-test.iudx.io` and port `80`.

Reply:
By remote host/port do you mean the proxy host/port?